editor/js/libs/ui.three.js

```diff
@@ -149,22 +178,17 @@ class UITexture extends UISpan {
 		if ( texture !== null ) {
 
 			const image = texture.image;
+			canvas.title = texture.sourceFile;
+			const scale = canvas.width / image.width;
 
-			if ( image !== undefined && image !== null && image.width > 0 ) {
+			if ( image.data !== undefined || texture.isCompressedTexture ) {
```

Comment:
The entire image checks should be guarded by an additional `if` statement:
```js
if ( image !== undefined && image !== null ` ) {
```
A similar check was present in the original but has now been wrongly refactored.